handbook/marketing/content.md

```diff
@@ -9,8 +9,8 @@ At Sourcegraph, we define **content marketing** as generating awareness, buildin
 
 ## Quick links
 
-- [Editing blog posts](editing_blog_posts.md)
 - [Creating blog posts](creating_blog_posts.md)
+- [Editing blog posts](creating_blog_posts.md#editing-blog-posts)
```

Comment:
@content-hacker In the spirit of DRY, I have removed the dedicated page for editing blog posts and am instead linking to the existing section on the "Creating blog posts" page

Reply:
Good thinking!